enumfields/compat.py

```diff
@@ -65,3 +65,19 @@ def import_string(dotted_path):
         module_path, class_name = dotted_path.rsplit('.', 1)
         module = import_module(module_path)
         return getattr(module, class_name)
+
+
+def import_class(components):
```

Comment:
By the way @NovaDev94 , can you explain why this is necessary? I.e. in which case would the `enum` parameter to `EnumField` be a sequence of strings?